src/ini_to_core_reader.cpp

```diff
@@ -110,14 +125,37 @@ int INIToCoreReader::valueHandler( void* user, const char* section, const char*
             // remove the special case identifier to figure out the actual file name
             // to process
             string csvFileName( valueStr.begin() + csvFilePrefix.size(), valueStr.end() );
-            // when not an absolute path consider the CSV filepath to be
-            // relative to the INI file
+            #ifdef USE_RCPP
+            // ANS: This is the algorithm used if Hector is compiled
+	    // as an R package.
+            //
+	    // If the csvFileName normalizes to a real path, use that.
+            // Otherwise, assume that it is pointing to a file in the
+            // same directory as the INI file.
+            try {
+                // Second argument is "winslash". "\\" (single forward
```

Comment:
```suggestion
                // Second argument is "winslash". "\\" (single backward
```
Is it worth noting here that all this argument does is to specify what the path separator is on Windows, and that it doesn't do anything on non-Windows platforms?  Reading this, I was a little concerned that we were specifying backslash as the default path separator, and had to go look up the docs for `normalizePath` to see what was up.  Then again, this code will likely never get touched again, so maybe it's not worth fussing with.

Reply:
Addressed in 09ade41.